src/main/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImpl.java

```diff
@@ -23,6 +23,17 @@ public class PnUserAttributesClientImpl {
     private final CourtesyApi courtesyApi;
     private final LegalApi legalApi;
 
+    public Mono<Void> acceptConsentPg(String xPagopaPnUid, CxTypeAuthFleet xPagopaPnCxType,
+                                    ConsentType consentType, ConsentAction consentAction, String version) {
+        log.logInvokingExternalService(PnLogger.EXTERNAL_SERVICES.PN_USER_ATTRIBUTES, "acceptConsentPg");
```

Comment:
here we log the service that we are invoking, so it is setPgConsentAction

Reply:
fix